src/monitor.py

```diff
@@ -1,14 +1,13 @@
-import json
-import time
+from typing import Callable, Iterable
 
-from typing import Callable, Iterable, Any
+from health import check_until_job_is_operational
 
 from kubernetes import client
 from kubernetes.client import V1ObjectMeta, V1PodStatus, ApiException, V1ContainerStatus
 
 from lifecycle.config import Config
 from lifecycle.monitor.base import JobMonitor
-from lifecycle.monitor.health import check_until_job_is_operational, quick_check_job_condition
+from lifecycle.monitor.health import quick_check_job_condition
```

Comment:
Don't you want to also use vendored `quick_check_job_condition`?

Reply:
will do